YoutubeExplode/Bridge/PlaylistExtractor.cs

```diff
@@ -106,60 +131,80 @@ internal partial class PlaylistExtractor
             .EnumerateArrayOrNull()?
             .Select(j => new ThumbnailExtractor(j))
             .ToArray() ??
+        
+        TryGetSidebarPrimary()?
+                .GetPropertyOrNull("thumbnailRenderer")?
+                .GetPropertyOrNull("playlistCustomThumbnailRenderer")?
+                .GetPropertyOrNull("thumbnail")?
+                .GetPropertyOrNull("thumbnails")?
+                .EnumerateArrayOrNull()?
+                .Select(j => new ThumbnailExtractor(j))
+                .ToArray() ??
+
+        //next endpoint part
+        GetVideos()
+            .FirstOrDefault()?
+            .GetVideoThumbnails()
+            .ToArray() ??
 
         Array.Empty<ThumbnailExtractor>()
     );
 
-    private JsonElement? TryGetContentRoot() => Memo.Cache(this, () =>
-        // Initial response
-        _content
-            .GetPropertyOrNull("contents")?
-            .GetPropertyOrNull("twoColumnBrowseResultsRenderer")?
-            .GetPropertyOrNull("tabs")?
-            .EnumerateArrayOrNull()?
-            .ElementAtOrNull(0)?
-            .GetPropertyOrNull("tabRenderer")?
-            .GetPropertyOrNull("content")?
-            .GetPropertyOrNull("sectionListRenderer")?
-            .GetPropertyOrNull("contents")?
-            .EnumerateArrayOrNull()?
-            .ElementAtOrNull(0)?
-            .GetPropertyOrNull("itemSectionRenderer")?
-            .GetPropertyOrNull("contents")?
-            .EnumerateArrayOrNull()?
-            .ElementAtOrNull(0)?
-            .GetPropertyOrNull("playlistVideoListRenderer")?
-            .GetPropertyOrNull("contents") ??
-
-        // Continuation response
-        _content
-            .GetPropertyOrNull("onResponseReceivedActions")?
-            .EnumerateArrayOrNull()?
-            .ElementAtOrNull(0)?
-            .GetPropertyOrNull("appendContinuationItemsAction")?
-            .GetPropertyOrNull("continuationItems")
+    public string? TryGetPlayListUrl() => Memo.Cache(this, () =>
+     TryGetPlaylistProperty()?
+         .GetPropertyOrNull("playlistShareUrl")?
```

Comment:
Does this work for both old and new approaches?

Reply:
playlistShareUrl always has this format ```https://www.youtube.com/watch?v=ID&list=ID```, this method does only work with next endpoint responses and next endpoint is used as a fallback of browse enpoint when trying to fetch metadata for mixes playlists.

Reply:
Do we actually need to extract this considering we already know the first video ID? We can just create the URL ourselves and not rely on `platlistShareUrl` being in the payload.